### PR TITLE
Classifier

### PR DESCRIPTION
**Pull request by cskiz**
Addressed previous comments.

### CODE REVIEW

**Review by cWren0110 — CHANGES_REQUESTED** [review]
Please replace `dataScience.src.<...>` references with `gamechangerml.src.<...>`

**Comment by cskiz** [comment]
I updated `table.py` and `ref_list.py`.

**Review by cWren0110 — CHANGES_REQUESTED** [review]

File: gamechangerml/src/text_classif/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+matplotlib==3.3.0
```
Comment (cWren0110) [inline gamechangerml/src/text_classif/requirements.txt:1]: Please add/resolve these in `gc-venv-green.txt` in the base of the repository.

**Review by cWren0110 — CHANGES_REQUESTED** [review]
Last change required.  Once made will Approve.

**Comment by cskiz** [comment]
`requirements.txt` has been deleted as redundant.

**Review by cWren0110 — APPROVED** [review]

**Comment by cWren0110** [comment]
@rha930 @jgrundy Please review when you have a chance.  Thank you.

**Comment by rha930** [comment]
my review mainly is for the structure so i trust everything else is OK

**Review by rha930 — APPROVED** [review]

**Pull request merged**